### DISCUSSION TITLE
Transportable skills

### DISCUSSION THREAD

**Discussion by jrub** [getmanfred/mac#21] [Q&A] [answered]
I've just been looking around at your product roadmap and your open source code, and well, congrats!! 👏

I was wondering about the use of https://github.com/getmanfred/mac/blob/master/skills/transportable_skills_list.md
Is this something actively used to validate something?

**Comment by dbonillaf** [comment, marked as answer]
Hi @jrub! Yup, the idea is to collect as many soft skills as possible to model it in our database. Instead of using just free text (a nightmare for searching and matching purposes), our bet is to model as many as possible data items. 

If you want to contribute to the list, it would be awesome!

**Reply by jrub** [reply]
Thanks for explaining!

I'd love to contribute but it's not something like I have 100% clear to be able to distill on a commit. Perhaps some short discussion here could help?

On one side, I would remove or rewrite any skill from the list that does not match my own principles for healthy and sustainable work. Here I fit `meet deadlines`, `results oriented`, `write requirements`, `report information`, `supervise others` , `take inventory`, `typing`. For some of these, I would try to use others instead, like `adapt to change`, `outcome oriented`, `understand the business`, `lead servantly`, `collaborate fully asynchronously`... 

There are others that I miss, but I would need help stating it on a way everyone understands, for instance: `comfortable being the glue not the medal`, `read early signals from patterns`, `strategic thinking`... I may add more later, this is just a quick "dump-thought".

Also, I would unify how they're all expressed. Like, verb or gerund? (adapts to change vs adapting to change), Camel Case or just one mayus? etc
